src/test/java/com/karankumar/bookproject/backend/utils/PredefinedShelfUtilsTest.java

```diff
@@ -0,0 +1,126 @@
+package com.karankumar.bookproject.backend.utils;
+
+import com.karankumar.bookproject.annotations.IntegrationTest;
+import com.karankumar.bookproject.backend.entity.Author;
+import com.karankumar.bookproject.backend.entity.Book;
+import com.karankumar.bookproject.backend.entity.PredefinedShelf;
+import com.karankumar.bookproject.backend.repository.BookRepository;
+import com.karankumar.bookproject.backend.repository.PredefinedShelfRepository;
+import com.karankumar.bookproject.backend.service.PredefinedShelfService;
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.springframework.beans.factory.annotation.Autowired;
+
+import java.util.List;
+import java.util.Set;
+
+import static com.karankumar.bookproject.backend.utils.ShelfUtils.ALL_BOOKS_SHELF;
+
+
+@IntegrationTest
+public class PredefinedShelfUtilsTest {
+
+    @Autowired
+    private PredefinedShelfRepository shelfRepository;
+
+    @Autowired
+    private BookRepository bookRepository;
+
+    private static PredefinedShelfUtils predefinedShelfUtils;
+
+    private static PredefinedShelf toReadShelf;
+    private static PredefinedShelf readingShelf;
+    private static PredefinedShelf readShelf;
+    private static PredefinedShelf didNotFinishShelf;
+    private static PredefinedShelf allShelves;
```

Comment:
I don't think this variable is used at all, so it could be removed

Reply:
done